mappings/net/minecraft/class_9226.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_9226
+	FIELD field_49035 PACKET_CODEC Lnet/minecraft/class_9139;
+	FIELD field_49036 DEFAULT_NAMESPACE Ljava/lang/String;
+	METHOD method_56931 isVanilla ()Z
+	METHOD method_56932 (Ljava/lang/String;)Lnet/minecraft/class_9226;
+		ARG 0 id
```

Comment:
I was calling this path based on how Identifier calls it path instead of id.  Thoughts?

Reply:
Also I ended  up calling this class DatapackIdentifier, since its the same as what a normal Identifier has but also contains a version string.

Reply:
Did end up calling it `VersionedIdentifier` and made the changes you suggested.
I also mapped one more method as #createVanilla as it creates the class with the vanilla namespace (minecraft)